unit/goto-programs/goto_program_validate.cpp

```diff
@@ -0,0 +1,429 @@
+/*******************************************************************\
+
+ Module: Unit tests for goto program validation
```

Comment:
Nit pick: no space before Module, Author, Date

Reply:
Done. Elsewhere too.

Reply:
Needs to be addressed again.